src/main/java/com/github/hcsp/encapsulation/Student.java

```diff
@@ -10,4 +10,38 @@
 
     /** 分数 */
     private int score;
+
+    private boolean fail;
+
+    public boolean isFail() {
+        return fail;
+    }
+
+    public void setFail(boolean fail) {
+        this.fail = fail;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public boolean isRetakingExam() {
+        return retakingExam;
+    }
+
+    public void setRetakingExam(boolean retakingExam) {
+        this.retakingExam = retakingExam;
+    }
+
+    public int getScore() {
+        return score;
+    }
+
+    public void setScore(int score) {
```

Comment:
```suggestion
    public void setScore(int score) {
        this.fail = score < 60;
```

否则会出现状态不一致的情况：score小于60但是fail=false。

另外，更好的解决方案是直接抛弃fail这个属性，用一个getter代替：

```
    public boolean isFail() {
        return score < 60;
    }
```